app/components/Settings/Preview/Preview.react.js

```diff
@@ -347,6 +348,7 @@ class Preview extends Component {
 
                                             dialect={dialect}
                                             runQuery={this.runQuery}
+                                            scheduleQuery={this.props.openScheduler}
```

Comment:
is it possible to rename this property to `openScheduler` since this seems to be the name use elsewhere?